cmd/koord-manager/main.go

```diff
@@ -78,10 +80,16 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+var controllers = []string{
```

Comment:
Here we can do a little optimization
```suggestion
var controllerAddFuncs = map[string]func(manager.Manager) error{
	nodemetric.Name:   nodemetric.Add,
	noderesource.Name: noderesource.Add,
	nodeslo.Name:      nodeslo.Add,
}

var controllers = set.StringKeySet(controllerAddFuncs)
```